services/training/crafting-swift-mobile-apps.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: training
+training: Test driving mobile apps
+---
+
+<!-- Course summary -->
+<div class="row text-center margin-bottom-60">
+    <div class="col-md-6 col-md-offset-3">
+        <h1>Crafting Swift Mobile Apps</h1>
+        <h4 class="margin-bottom-20">Created by Ana Nogal</h4>
+        <p class="margin-bottom-40 training-description"><strong>For experienced mobile developers that want to give the next step with their apps and not only build beautiful apps, but also reliable apps.</strong>
+          Learn how to TDD your app using Outside-in approach.
+        </p>
+        <p><strong>This course is also available in Spanish.</strong></p>
+        <a href="/services#training" class="training-link-btn btn-u btn-brd btn-u-orange btn-u-lg margin-right-20"
+           type="button">More courses</a>
+        <a href="#contact" class="training-link-btn btn-u btn-brd btn-u-orange btn-u-lg" type="button">Contact</a>
+    </div>
+</div>
+<!-- End course summary -->
+
+<!-- Course description -->
+<div class="row">
+    <div class="col-md-8 col-md-offset-2">
+        <div class="row margin-bottom-20">
+            <h3>Course Duration</h3>
+	    <p>2 days</p>
+        </div>
+
+	<div class="row margin-bottom-20">
+            <h3>Why attend?</h3>
+            <ul>
+                <li>  Learn how SOLID principles and TDD can help you build code that is easy to add new features.</li>
```

Comment:
I'm not sure if 'build code' is the correct choice of words.
'Build apps' or 'create code' or 'write code' might be better alternatives.

'... can help you build code that is easy to add new features' does not make sense.

Another alternative: 

Learn how SOLID principles and TDD can help you add new features to your code base with less effort.